templates/deployment.yaml

```diff
@@ -103,6 +103,16 @@ spec:
           mountPath: /opt/Actian/Actian_Client
         - name: datacloud-jdbc-avalanche-vol
           mountPath: /opt/Actian/lib/jdbc
+        - name: datacloud-odbc-redshift-vol
```

Comment:
I tried to install all odbc drivers into a single shared volume, but seems not possible as we are using the vendor specific instructions and yum install does not allow to change the directory.